CHANGELOG.md

```diff
@@ -40,6 +40,17 @@ All notable changes to this project will be documented in this file.
 ## Unreleased
 * ### ALL
     * #### Added
+        * Added docs_static/nirfsg_*.inc files for nirfsg 
```

Comment:
We like to limit the changelog to user-facing behavioral changes.
Low-level template tweaks and documentation changes generally shouldn't be mentioned.